lib/take_away.rb

```diff
@@ -0,0 +1,54 @@
+
+class TakeAway
+  attr_reader :order, :dish_hash
+  def initialize
+    @order = {}
+    @dish_hash = {}
+    #create_dish_hash
+  end
+
+  def show_menu
+    # can be refactored to take menu.csv as input file
+    filename = "lib/menu.txt"
+    File.open filename do |f|
+      f.each { |line| puts line }
+    end
+  end
+
+  def enter_order
+    # can be refactored to use one method with an
+    # puts and a gets statement
+    # and an abstracted method to use variables for
+    # the message, the input variable and an validation
+    puts "Enter the dish number and the quantity when ask"
+    loop do
+      puts "Enter a dish number or 99 to end ordering"
+      dish_number = gets.chomp
+      break if dish_number.to_i == 99
+      if dish_number.to_i <= 0 or dish_number.to_i > 19
```

Comment:
Style/AndOr: Use `||` instead of `or`. (https://github.com/bbatsov/ruby-style-guide#no-and-or-or)